ocp_utilities/utils.py

```diff
@@ -16,7 +16,7 @@ def run_command(
     shell=False,
     timeout=None,
     capture_output=True,
-    check=True,
+    check=False,
```

Comment:
please do not change this here, change the caller when needed

Reply:
Ok, thanks